public/app/screens/main/sections/devices/device-detail/img/close.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 17 17" enable-background="new 0 0 17 17" xml:space="preserve">
+<g>
+	<line fill="none" stroke="#575756" stroke-miterlimit="10" x1="0.5" y1="0.5" x2="16.5" y2="16.5" stroke-width="1.2"/>
+	<line fill="none" stroke="#575756" stroke-miterlimit="10" x1="0.5" y1="16.5" x2="16.5" y2="0.5" stroke-width="1.2"/>
+</g>
+</svg>
```

Comment:
Does webpack automatically recognize duplicate resources? The same resource is being used for the device finder as well